lib/Runtime/Language/SourceTextModuleRecord.h

```diff
@@ -50,6 +50,10 @@ namespace Js
         Var GetSpecifier() const { return normalizedSpecifier; }
         const char16 *GetSpecifierSz() const { return JavascriptString::FromVar(this->normalizedSpecifier)->GetSz(); }
 
+        void SetModuleURL(Var moduleURL) {this->moduleURL = moduleURL;}
```

Comment:
These small in-line methods like this should have spaces padding the body from the curly-braces. (`{this}` => `{ this }`)